src/commands/utility/search-code-review.js

```diff
@@ -0,0 +1,74 @@
+const { SlashCommandBuilder, ChannelType } = require('discord.js');
+const { translateLanguage } = require('../../languages/index');
+const { MAPPED_STATUS_COMMANDS } = require('../../config');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('check-review')
+    .setDescription(translateLanguage('checkReview.description'))
+    .addChannelOption((option) =>
+      option
+        .setName('channel')
+        .setDescription(translateLanguage('checkReview.channelOption'))
+        .addChannelTypes(ChannelType.GuildText)
+        .setRequired(true)
+    ),
+
+  async execute(interaction) {
+    try {
+      const channel = interaction.options.getChannel('channel');
+      if (!channel) {
+        await interaction.reply({
+          content: translateLanguage('checkReview.noChannels'),
+          ephemeral: true,
+        });
+        return;
+      }
+
+      const statusText = MAPPED_STATUS_COMMANDS['pr-request-review'];
+      if (!statusText) {
+        throw new Error('Mapped status for pr-request-review not found.');
+      }
```

Comment:
You can use the function that you declared here: https://github.com/TeamNovaSoft/discord-bot/pull/89/files#diff-0e096a2d7da3d0417d7bf9a41f9092bc8ac20010e7549146ad23e95bc479b659R18-R24